phy/cluster/algorithms.py

```diff
@@ -155,7 +164,7 @@ def __init__(self, tempdir=None, probe=None, **kwargs):
     def _create_filter(self):
         rate = self._kwargs['sample_rate']
         low = self._kwargs['filter_low']
```

Comment:
Sorry, I may have missed something, are you doing away with the .low.kwd
file? I'm not talking about saving the LFP in the store, but rather in a
file like low.kwd.

Will add issue on specifying LFP filtering parameters. Is this a phy issue
now or spikedetekt?

On Tue, Jun 30, 2015 at 6:57 PM, Cyrille Rossant <notifications@github.com>
wrote:

> In phy/cluster/algorithms.py
> <https://github.com/kwikteam/phy/pull/448#discussion_r33602966>:
>
> > @@ -155,7 +164,7 @@ def __init__(self, tempdir=None, probe=None, **kwargs):
> >      def _create_filter(self):
> >          rate = self._kwargs['sample_rate']
> >          low = self._kwargs['filter_low']
>
> @nippoo <https://github.com/nippoo> I had misread your comment; yes these
> parameters have always been in Hertz!
>
> We can save the LFP, but in which format? We already save the HPF in
> temporary files (flat binary), one file per chunk (to avoid contention
> issues), shall we do the same for LFP too? In any case @nsteinme
> <https://github.com/nsteinme> please add an issue otherwise we'll forget
> it...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kwikteam/phy/pull/448/files#r33602966>.
>